tests/tests_suite_test.go

```diff
@@ -35,26 +37,59 @@ var _ = AfterSuite(func() {
 	removeTestNamespaces()
 })
 
+func getPodLogs(podName, containerName string) (string, error) {
+	req := testClient.VirtClient.CoreV1().Pods(managerNamespace).GetLogs(podName, &corev1.PodLogOptions{
+		Container: containerName,
+	})
+	podLogs, err := req.Stream(context.TODO())
+	if err != nil {
+		return "", err
+	}
+	defer func(podLogs io.ReadCloser) {
+		err := podLogs.Close()
+		if err != nil {
+			panic(err)
+		}
```

Comment:
do we need panic here ?
if we can refrain from it, it will be better please (maybe just log instead)

Reply:
DONE